custom/icds_reports/data_pull/data_pulls.py

```diff
@@ -193,3 +201,77 @@ def _dump_consolidated_data(result):
             row['State'] = state_name
             writer.writerow(row)
         return result_file
+
+
+class VHSNDMonthlyReport(MonthBasedDataPull):
+    slug = "vhsnd_monthly_report"
+    name = "VHSND monthly report"
+    queries = [
+        VHSNDMonthlyCount
+    ]
+
+    def file_format_required(self):
+        return 'xlsx'
+
+    def post_run(self, data_files):
+        result = self._consolidate_data(data_files)
+        state_results = self._format_consolidated_data(result)
```

Comment:
`_format_consolidated_data` can be called within `_consolidate_data` itself.

Reply:
changed